src/seedu/addressbook/AddressBook.java

```diff
@@ -812,6 +829,22 @@ private static boolean deletePersonFromAddressBook(HashMap<PersonProperty, Strin
     private static ArrayList<HashMap<PersonProperty, String>> getAllPersonsInAddressBook() {
         return ALL_PERSONS;
     }
+    
+    /**
+     * Returns all persons in the address book sorted alphabetically
+     */
+    private static ArrayList<HashMap<PersonProperty, String>> getAllPersonsInAddressBookSorted() {
```

Comment:
this method seems to be doing two things, try and extract out some stuff to improve SLAP on this method